fvm/fvm_test.go

```diff
@@ -1820,7 +1820,7 @@ func TestBlockContext_ExecuteTransaction_FailingTransactions(t *testing.T) {
 	)
 }
 func TestSigningWithTags(t *testing.T) {
-	t.Run("Signing Transactions without tag works (for now)", newVMTest().
+	t.Run("Signing Transactions without tag doesn't work", newVMTest().
```

Comment:
Do we also have tests with a tag, just for completeness?

Reply:
added